app/assets/javascripts/components/widgets_box.js.jsx

```diff
@@ -68,40 +68,54 @@ var Widget = React.createClass({
   },
 
   render(){
-    var rawMarkup = marked(this.props.children.toString(), {sanitize: true});
```

Comment:
I was just experimenting with third party libs. If we decide on markdown, we can always provide the campaigners with a markdown wysiwyg editor ( https://github.com/benweet/stackedit / https://stackedit.io/editor for example )